docs/source/notebooks/pcca.ipynb

```diff
@@ -75,7 +118,38 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "msm.transition_matrix.shape"
+    "import networkx as nx\n",
+    "\n",
+    "fig, axes = plt.subplots(1, 2, figsize=(15, 10))\n",
+    "\n",
+    "for i in range(len(axes)):\n",
+    "    threshold = 0 if i == 0 else 1e-2\n",
+    "    ax = axes[i]\n",
+    "    title = \"Transition matrix\"\n",
+    "    if i == 1:\n",
+    "        title += f\" with connectivity threshold {threshold:.0e}\"\n",
+    "    G = nx.DiGraph()\n",
+    "    ax.set_title(title)    \n",
+    "    for i in range(msm.n_states):\n",
+    "        G.add_node(i, title=f\"{i+1}\")\n",
+    "    for i in range(msm.n_states):\n",
+    "        for j in range(msm.n_states):\n",
+    "            if msm.transition_matrix[i, j] > threshold:\n",
+    "                G.add_edge(i, j, title=f\"{msm.transition_matrix[i, j]:.3e}\")\n",
+    "\n",
+    "    edge_labels = nx.get_edge_attributes(G, 'title')\n",
+    "    pos = nx.fruchterman_reingold_layout(G)\n",
+    "    nx.draw_networkx_nodes(G, pos, ax=ax)\n",
+    "    nx.draw_networkx_labels(G, pos, ax=ax, labels=nx.get_node_attributes(G, 'title'));\n",
+    "    nx.draw_networkx_edges(G, pos, ax=ax, arrowstyle='-|>',\n",
+    "                           connectionstyle='arc3, rad=0.3');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To make it rigorous, let us apply PCCA+ with two metastable sets:"
```

Comment:
we know that we should use 2 here, but maybe worth commenting on making this choice in general

Reply:
done